app/static/js/script.js

```diff
@@ -25,15 +23,6 @@
 			document.querySelectorAll('section').forEach((section) => {
 				section.classList.add('no-display')
 			});
-			
```

Comment:
All this code did was create errors. The class `lekkersicko` was never used in the css or in the js.